plugins/modules/system/java_keystore.py

```diff
@@ -11,68 +12,102 @@
 DOCUMENTATION = '''
 ---
 module: java_keystore
-short_description: Create or delete a Java keystore in JKS format.
+short_description: Create a Java keystore in JKS format
 description:
-     - Create or delete a Java keystore in JKS format for a given certificate.
+  - Bundle a x509 certificate and its private key into a Java Keystore in JKS format.
 options:
-    name:
-        type: str
-        description:
-          - Name of the certificate.
-        required: true
-    certificate:
-        type: str
-        description:
-          - Certificate that should be used to create the key store.
-        required: true
-    private_key:
-        type: str
-        description:
-          - Private key that should be used to create the key store.
-        required: true
-    private_key_passphrase:
-        description:
-          - Pass phrase for reading the private key, if required.
-        type: str
-        required: false
-        version_added: '0.2.0'
-    password:
-        type: str
-        description:
-          - Password that should be used to secure the key store.
-        required: true
-    dest:
-        type: path
-        description:
-          - Absolute path where the jks should be generated.
-        required: true
-    owner:
-        description:
-          - Name of the user that should own jks file.
-        required: false
-    group:
-        description:
-          - Name of the group that should own jks file.
-        required: false
-    mode:
-        description:
-          - Mode the file should be.
-        required: false
-    force:
-        description:
-          - Key store will be created even if it already exists.
-        required: false
-        type: bool
-        default: 'no'
-requirements: [openssl, keytool]
-author: Guillaume Grossetie (@Mogztter)
+  name:
+    description:
+      - Name of the certificate in the keystore.
+      - If the provided name does not exist in the keystore, the module fails.
+        This behavior will change in a next release.
+    type: str
+    required: true
+  certificate:
+    description:
+      - Content of the certificate used to create the keystore.
+      - If the fingerprint of the provided certificate does not match the
+        fingerprint of the certificate bundled in the keystore, the keystore
+        is regenerated with the provided certificate.
+      - One of I(certificate) or I(certificate_path) is required.
+      - I(certificate) and I(certificate_path) are mutually exclusive.
```

Comment:
How about:
```suggestion
      - Exactly one of I(certificate) or I(certificate_path) is required.
```
instead? (Same for the others.)

Reply:
done everywhere